### ISSUE TITLE
potential bug?

### ISSUE THREAD

**Issue by chijames** [gdamaskinos/unsupervised_topic_segmentation#2]
Hi,

Thanks for releasing the code.

Just wonder if we need to add textiling_hyperparameters.SENTENCE_COMPARISON_WINDOW to the predicted segments? The first possible segmentation should occur at textiling_hyperparameters.SENTENCE_COMPARISON_WINDOW, right?

Thanks.

**Comment by Akshayextreme** [comment]
yes, I also think the same. Looks like the code is not complete.
If first predicted segment is at `0` then the topic break should be at 
`(0 + extiling_hyperparameters.SENTENCE_COMPARISON_WINDOW)th` sentence in the original document

**Comment by aanchan** [comment]
@Akshayextreme I think it should be at 0 + extiling_hyperparameters.SENTENCE_COMPARISON_WINDOW + extiling.SMOOTHING_WINDOW, I am currently in the debugger and while the depth score computation removes SENTENCE_COMPARISON_WINDOW at either end, it removes extiling.SMOOTHING_WINDOW at either end as well

**Comment by completelyboofyblitzed** [comment]
@aanchan Thank you for participating in the issues, man
